### ISSUE TITLE
Salt and pepper Noise in Training Results

### ISSUE THREAD

**Issue by SurbhiKhushu** [junyanz/pytorch-CycleGAN-and-pix2pix#1150]
Hello,

I am trying to run on grey scale medical image dataset. In the start of the training I get salt and pepper noise which tends to reduce around epoch 40 ,but it is still visible in the generated data even in the 190th epoch.

Is there any possible solution to it and how do I figure out early in the training stage that it is a mode collapse just by the loss?

Best Regards

**Comment by junyanz** [comment]
I am not sure if your artifacts are caused by mode collapse. From the loss curve, the loss will become extremely small or large when mode collapse happens.

**Comment by zimbobwe** [comment]
I have the similar problem in my practice, and I think this may because I drop the normalization in image pre-processing.
